app_dart/lib/src/request_handlers/check_flaky_builders.dart

```diff
@@ -163,8 +164,9 @@ class CheckFlakyBuilders extends ApiRequestHandler<Body> {
   }
 
   bool _getIgnoreFlakiness(String? builderName, CiYaml ciYaml) {
-    final Target target = ciYaml.postsubmitTargets.singleWhere((Target target) => target.value.name == builderName);
-    return target.getIgnoreFlakiness();
+    final Target? target =
```

Comment:
Can we add a test to cover this?

Reply:
Do you have advice on a easy way to get to this function or spoof the data to trigger the issue? Need to have builderName be a value thats not found in the ciYaml object.

Reply:
an easy way is to make this function visible from test, and test this function specifically.

Reply:
added tests